.travis.yml

```diff
@@ -95,4 +95,4 @@ script:
     - if [[ $VALIDATE_TRANSLATION_FILE = '' ]]; then phpunit -v ; fi;
     - if [[ $CS_FIXER = run ]]; then php bin/php-cs-fixer fix src/ --verbose --dry-run ; fi;
     - if [[ $VALIDATE_TRANSLATION_FILE = run ]]; then php bin/console lint:yaml src/Wallabag/CoreBundle/Resources/translations -v ; fi;
-    - if [[ $ASSETS = build ]]; then ./node_modules/grunt-cli/bin/grunt; fi;
+    - if [[ $ASSETS = build ]]; then ./node_modules/grunt-cli/bin/grunt tests; fi;
```

Comment:
Indeed, there could be errors of js not building too. I want to see how long this takes already.